distributed/actor.py

```diff
@@ -166,16 +212,17 @@ async def run_actor_function_on_worker():
                             return await run_actor_function_on_worker()
                         else:
                             raise OSError("Unable to contact Actor's worker")
-                    return result
+                    if result["status"] == "OK":
+                        return _OK(result["result"])
+                    return _Error(result["exception"])
 
-                q = asyncio.Queue(loop=self._io_loop.asyncio_loop)
+                actor_future = ActorFuture(io_loop=self._io_loop)
 
                 async def wait_then_add_to_queue():
```

Comment:
There is no queue anymore, is there? This could be renamed

Reply:
yep!